nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ClusterCommandHelper.java

```diff
@@ -436,4 +430,12 @@ private static String listToString(List<String> slist) {
         public final List<String> succeededServerNames = new ArrayList<>();
         public final List<String> failedServerNames = new ArrayList<>();
     }
+
+    /**
+     * Set the timeout for ClusterCommandHelper
+     * @param adminTimeout in milliseconds
+     */
+    public void setAdminTimeout(long adminTimeout) {
+        this.adminTimeout = adminTimeout;
```

Comment:
Shouldn't this be "_timeout - 3 seconds_" (if timeout is greater than 3 seconds obviously)?
I'm wary of removing of removing this "wiggle room" that's been present for years.